lib/jekyll/scholar/utilities.rb

```diff
@@ -135,10 +141,6 @@ def entries
         sort bibliography[query || config['query']]
       end
 
-      def limit_entries?
```

Comment:
Or rather, I would prefer if max returns either nil or, if set, the correct value (taking into account the offset). That way you don't have to set max specifically inside the Bibliography tag.